neuro_san/service/agent_main_loop.py

```diff
@@ -137,6 +145,12 @@ def main_loop(self):
                                   request_limit=self.request_limit,
                                   forwarded_request_metadata=self.forwarded_request_metadata)
 
+        if self.manifest_update_period > 0:
+            manifest_file: str = self.manifest_files[0]
+            updater: ManifestPeriodicUpdater =\
+                ManifestPeriodicUpdater(manifest_file, self.manifest_update_period)
+            updater.start()
+
```

Comment:
Start manifest updater if requested.

Reply:
It feels like there is something else that needs to happen in the http server so it knows if there is a new agent to serve or an old agent to take down from an HTTP standpoint.  Is that already laid down as part of previous checkins?

Reply:
Yes.